src/router.ts

```diff
@@ -0,0 +1,21 @@
+import { DIPDeposit } from "./common";
+
+export class Router {
+  mm_callback: (d: DIPDeposit) => void;
+  risk_manager_callback: (d: DIPDeposit) => void;
+
+  constructor(
+    mm_callback: (d: DIPDeposit) => void,
+    risk_manager_callback: (d: DIPDeposit) => void
+  ) {
+    this.mm_callback = mm_callback;
+    this.risk_manager_callback = risk_manager_callback;
+  }
+
+  // Accepts a DIP Deposit and decides whether to send it to the mm_callback
+  // or risk_manager_callback
+  route(dip_deposit: DIPDeposit) {
+    // TODO: Decide where to send the deposit to.
+    this.mm_callback(dip_deposit);
```

Comment:
everything getting sent to the MM right now?

Reply:
typo, will fix when connecting the scalper